### ISSUE TITLE
Exec.exec fails on json.parse

### ISSUE THREAD

**Issue by GiladShoham** [kubernetes-client/javascript#170]
Using version 0.7.1 when I run exec which fails (status code 1) I got a valid json on the handler - https://github.com/kubernetes-client/javascript/blob/master/src/web-socket-handler.ts#L47
So the JSON.parse is working as expected. 
Since version 0.7.2 I got a string which  make the JSON.parse to throw.
Here is real example of response from 0.7.1 and 0.7.2
0.7.1:
```js
{
	"metadata": {},
	"status": "Failure",
	"message": "command terminated with non-zero exit code: Error executing in Docker Container: 137",
	"reason": "NonZeroExitCode",
	"details": {
		"causes": [{
			"reason": "ExitCode",
			"message": "137"
		}]
	}
}
```
0.7.2:
```js
"command terminated with non-zero exit code: Error executing in Docker Container: 137"
```

**Comment by brendandburns** [comment]
I'll investigate this.

**Comment by brendandburns** [comment]
If I remember correctly, this has something to do with the various different websockets protocols, though I don't know what changed between releases...

**Comment by brendandburns** [comment]
Ah, I figured it out, it was in the switch to isomorphic-ws we dropped the protocols header....

I'll see about fixing.

**Comment by brendandburns** [comment]
PR with the fix here: #180

Once this merges I'll cut a new release.

Apologies for the oversight during the refactor....
